sim/midas/src/main/cc/bridges/clock.h

```diff
@@ -43,6 +45,20 @@ class clockmodule_t final : public widget_t {
    */
   uint64_t hcycle();
 
+  /**
+   * Credit the system to deliver a given number of clock ticks to the target.
```

Comment:
This needs to be updated. Here's what i'm thinking. Potentially link out to this: https://docs.fires.im/en/stable/Advanced-Usage/Generating-Different-Targets.html?highlight=base%20clock#the-base-clock

```suggestion
   * Credits the bridge, allowing the simulation to advance up to <credits> more base clock cycles.
   * 
   * Note, if a total of N credits have been granted, no intermediate positive edges in other, out-of-phase clock domains will be simulated.  In other words. the simulation will halt after the Nth positive edge in the base clock domain + a delta in which combinational logic driven by that launching edge has been resolved. 
   */
```

Reply:
Will need to be formatted.